data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/impl/ReceiverVerticle.java

```diff
@@ -235,8 +233,6 @@ public void handle(HttpServerRequest request) {
     public void updateServerConfig() {
 
         // This function will be called when the secret volume is updated
-        File tlsKeyFile = new File(tlsKeyFilePath);
-        File tlsCrtFile = new File(tlsCrtFilePath);
 
         // Check whether the tls.key and tls.crt files exist
         if (tlsKeyFile.exists() && tlsCrtFile.exists() && httpsServerOptions != null) {
```

Comment:
On line 253, because `tlsCrtFilePath` is not a global variable, you cannot use it anymore. Instead use `tlsCrtFile.toPath()`